### DISCUSSION TITLE
0.0.15

### DISCUSSION THREAD

**Discussion by Hellowlol** [custom-components/nordpool#414] [Announcements]
## What's Changed
* Update VAT for Belgium by @frafra in https://github.com/custom-components/nordpool/pull/337
* Fix issue with late published prices by @viiru- in https://github.com/custom-components/nordpool/pull/333
* Pass areas down to aio_price.py to avoid errors unrelated to selected area by @Newspaperman57 in https://github.com/custom-components/nordpool/pull/341
* Update README.md by @PhPersson in https://github.com/custom-components/nordpool/pull/344
* Update Estonian VAT by @rixxxx in https://github.com/custom-components/nordpool/pull/364
* Fix blocking calls inside the event loop by @rrooggiieerr in https://github.com/custom-components/nordpool/pull/383
* Reverse order of operations to properly ignore DST issues in data by @viiru- in https://github.com/custom-components/nordpool/pull/375
* Add Latvian translation by @avalanche9 in https://github.com/custom-components/nordpool/pull/374
* Fixes non async call async_forward_entry_setup by @rrooggiieerr in https://github.com/custom-components/nordpool/pull/390
* Remove unnecessary call to template.attach by @emontnemery in https://github.com/custom-components/nordpool/pull/397
* Update Finnish VAT by @vla89 in https://github.com/custom-components/nordpool/pull/394
* Update aio_price.py by @Hellowlol in https://github.com/custom-components/nordpool/pull/400
* Update __init__.py by @Hellowlol in https://github.com/custom-components/nordpool/pull/401
* Revert by @Hellowlol in https://github.com/custom-components/nordpool/pull/409
* Create sk.json by @misa1515 in https://github.com/custom-components/nordpool/pull/359
# Breaking changes
* New API integration by @developerfromjokela in https://github.com/custom-components/nordpool/pull/413

## New Contributors
* @frafra made their first contribution in https://github.com/custom-components/nordpool/pull/337
* @viiru- made their first contribution in https://github.com/custom-components/nordpool/pull/333
* @Newspaperman57 made their first contribution in https://github.com/custom-components/nordpool/pull/341
* @PhPersson made their first contribution in https://github.com/custom-components/nordpool/pull/344
* @rixxxx made their first contribution in https://github.com/custom-components/nordpool/pull/364
* @rrooggiieerr made their first contribution in https://github.com/custom-components/nordpool/pull/383
* @avalanche9 made their first contribution in https://github.com/custom-components/nordpool/pull/374
* @emontnemery made their first contribution in https://github.com/custom-components/nordpool/pull/397
* @vla89 made their first contribution in https://github.com/custom-components/nordpool/pull/394
* @misa1515 made their first contribution in https://github.com/custom-components/nordpool/pull/359
* @developerfromjokela made their first contribution in https://github.com/custom-components/nordpool/pull/413
* 

**Full Changelog**: https://github.com/custom-components/nordpool/compare/0.0.14...0.0.15

<hr /><em>This discussion was created from the release <a href='https://github.com/custom-components/nordpool/releases/tag/0.0.15'>0.0.15</a>.</em>

**Comment by oywino** [comment]
I'm currently running 0.0.14
Why isn't an update to 0.0.15 being initiated automatically?

**Comment by imist** [comment]
Any description of the breaking api change? Do we have to change anything when updating?

**Reply by developerfromjokela** [reply]
> Any description of the breaking api change? Do we have to change anything when updating?

Check region codes, instead of Oslo it is now NOX (NO1, NO2, etc).

**Comment by Hellowlol** [comment]
Norwegian users needs to delete the/updste yaml due to zone name change. Others users might see thet new devices has been made

**Reply by oywino** [reply]
wat does "the/updste" mean?

**Reply by apetrynet** [reply]
_Update_

**Comment by typxxi** [comment]
Time to say thank you cause it works right now properly, I am back to 0,0x cent deviation for german tibber usage. 
And that is caused mainly by rounded input figures I get from Tibber and then it changes. 

Hats off, great job to guide me through and of cause I hope that you will change the german vat from 0.23 which is and was false to 0.19

RIght now I am working with a sensor that does not inclued taxes, so I do it on my own and can live better with that pure sensor values cause those are easier to crosscheck with nordpool and entso-e.
